pkgs/servers/guacamole-server/default.nix

```
@@ -0,0 +1,95 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, autoPatchelfHook
+, autoreconfHook
+, cairo
+, ffmpeg_4-headless
+, freerdp
+, libjpeg_turbo
+, libpng
+, libossp_uuid
+, libpulseaudio
+, libssh2
+, libtelnet
+, libvncserver
+, libvorbis
+, libwebp
+, libwebsockets
+, makeBinaryWrapper
+, openssl
+, pango
+, perl
+, nixosTests
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "guacamole";
+  version = "1.5.2";
+
+  src = fetchFromGitHub {
+    owner = "apache";
+    repo = "guacamole-server";
+    rev = finalAttrs.version;
+    hash = "sha256-L1hFZ24kwTSHwqCUslnt5cBKkNh1cpVxu1ntTN1gFr0=";
+  };
+
+  NIX_CFLAGS_COMPILE = [
+    "-Wno-error=format-truncation"
+    "-Wno-error=format-overflow"
+  ];
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    autoreconfHook
+    cairo
+    ffmpeg_4-headless
+    freerdp
+    libjpeg_turbo
+    libpng
+    libossp_uuid
+    libpulseaudio
+    libssh2
+    libtelnet
+    libvncserver
+    libvorbis
+    libwebp
+    libwebsockets
+    makeBinaryWrapper
+    openssl
+    pango
+    perl
+    pkg-config
+  ];
+
+  configureFlags = [
+    ''--with-freerdp-plugin-dir=${placeholder "out"}/lib''
```

```suggestion
    "--with-freerdp-plugin-dir=${placeholder "out"}/lib"
```
those are nested inside the ${}